src/main/app/src/app/zaken/model/zaak-ontkoppel-gegevens.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {ZaakRelatietype} from './zaak-relatietype';
+
+export class ZaakOntkoppelGegevens {
+    bronZaakUuid: string;
```

Comment:
Wat wel en niet duidelijk is in naamgeving is subjectief. De term 'bronZaak' geeft mij geen indicatie of dit nou de zaak is die ontkoppelt moet worden of de zaak waarvan een zaak ontkoppelt wordt. Ik zou bronZaakUUID teOntkoppelenZaakUUID noemen en identificatie ontkoppelenVanZaakIdentificatie. Maar leg het anders ook eens voor aan de andere developpers

Reply:
Heb het aangepast, zat ook te zoeken naar een juiste naamgeving, denk dat jouwn voorstel inderdaad beter is.